libs/ledger-live-common/src/exchange/index.ts

```diff
@@ -11,7 +11,7 @@ import { findTestExchangeCurrencyConfig } from "./testCurrencyConfig";
 const exchangeSupportAppVersions = {
   bitcoin_cash: "1.5.0",
   bitcoin_gold: "1.5.0",
-  bitcoin: "1.5.0",
+  bitcoin: "2.1.0",
```

Comment:
No need to update the version for the other ones than bitcoin?

Reply:
in fact, it doesn't matter as the exchange app 2.0.12 supports both old and new altcoin apps well. old and new altcoin app use the same APIs.
but anyway, as I've updated the required versions in support.ts. I update the versions for all the other ones now.